numba/tests/test_parfors.py

```diff
@@ -2251,6 +2251,28 @@ def test_impl(arr, l, flag):
         self.check(test_impl, arr, 10, True)
         self.check(test_impl, arr, 10, False)
 
+    def test_issue8515(self):
+        """ issue8515: an array is filled in the first prange and
+            then accessed with c[i - 1] in the next prange which
+            should prevent fusion with the previous prange.
+        """
```

Comment:
```suggestion
        # issue8515: an array is filled in the first prange and
        # then accessed with c[i - 1] in the next prange which
        # should prevent fusion with the previous prange.
```
suggest this else the docstring appears in the test logs which makes them harder to grep.